doc/source/intro.rst

```diff
@@ -13,15 +13,17 @@ The object database implementation is optimized for handling large quantities of
 Requirements
 ============
 
-* `Python`_ >= 3.6
+* `Python`_ >= 3.7
 * `Git`_ 1.7.0 or newer
     It should also work with older versions, but it may be that some operations
     involving remotes will not work as expected.
 * `GitDB`_ - a pure python git database implementation
+* `typing_extensions`_ >= 3.10.0
```

Comment:
According to requirements.txt this should be "typing-extensions>=3.7.4.3;python_version<"3.10""